pyrsia_node/src/main.rs

```diff
@@ -150,7 +150,7 @@ fn setup_pyrsia_services(
     let artifact_path = PathBuf::from(ARTIFACTS_DIR.as_str());
     let (build_event_sender, build_event_receiver) = mpsc::channel(32);
 
-    debug!("Create artifact service");
+    debug!("Create artifact service: ARTIFACTS_DIR = {}", artifact_path.display());
```

Comment:
"cargo fmt -- --check" reports errors
```suggestion
    debug!(
        "Create artifact service: ARTIFACTS_DIR = {}",
        artifact_path.display()
    );
```

Reply:
done. amended PR